src/com/service/PersonService.java

```diff
@@ -0,0 +1,78 @@
+package com.service;
+
+import com.person.Person;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class PersonService implements IService{
+
+    private List<Person> allPersons;
+
+    public PersonService() {
+        this.allPersons = new ArrayList<>();
+    }
+
+    private boolean findPersonByDateOfBirth(Date dateOfBirth) {
```

Comment:
Why boolean ? If you want to use boolean the naming convention is "isSomethindSomething". Here if you want to return boolean you should name it isPersonAlreadyCreatedByDateOfBirth or something like it. If you want to stay like this make it return Person

Reply:
Thanks again for your explanation. Fixed that already